mmv1/products/dialogflowcx/terraform.yaml

```diff
@@ -0,0 +1,41 @@
+# Copyright 2020 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Provider::Terraform::Config
+overrides: !ruby/object:Overrides::ResourceOverrides
+  Agent: !ruby/object:Overrides::Terraform::ResourceOverride
+    timeouts: !ruby/object:Api::Timeouts
+      insert_minutes: 40
+      update_minutes: 40
+    examples:
+      - !ruby/object:Provider::Terraform::Examples
+        name: "dialogflowcx_agent_full"
+        primary_resource_id: "full_agent"
+        vars:
+          agent_name: "dialogflowcx-agent"
+    id_format: "{{name}}"
```

Comment:
We'll want to revert both of `id_format` and `name`, they should take on the correct value automatically. If not, they're both `projects/{{project}}/locations/{{location}}/agents/{{name}}`